frontend/packages/kubevirt-plugin/src/k8s/requests/vmi/objects/migration.ts

```diff
@@ -0,0 +1,37 @@
+import { getName, getNamespace } from '@console/shared';
+import { apiVersionForModel, K8sResourceKind } from '@console/internal/module/k8s';
+import { VMIKind } from '../../../../types/vm';
+import { VirtualMachineInstanceMigrationModel } from '../../../../models';
+
+export class Migration {
+  data: K8sResourceKind;
```

Comment:
Nit: in TypeScript, class members are public by default. I know it's a small thing, but I'd suggest to mark `data` as `private`, unless we expect other code to access it directly.